Tests/GRPCNIOTransportHTTP2Tests/HTTP2TransportTLSEnabledTests.swift

```diff
@@ -185,6 +187,148 @@ struct HTTP2TransportTLSEnabledTests {
     }
   }
 
+  @Test("Custom certification callbacks are used for verification.")
+  @available(gRPCSwiftNIOTransport 2.0, *)
+  func testRPC_mTLS_customVerificationCallback_OK() async throws {
+    // Create a new certificate chain that has 4 certificate/key pairs: root, intermediate, client, server
+    let certificateChain = try CertificateChain()
+    let filePaths = try certificateChain.writeToTemp()
+    let clientConfig = self.makeMTLSClientConfig(
+      certificatePath: filePaths.clientCert,
+      keyPath: filePaths.clientKey,
+      trustRootsPath: filePaths.trustRoots,
+      serverHostname: CertificateChain.serverName
+    )
+    let expectedCertificates = [certificateChain.client.certificate]
+    try await confirmation { confirmation in
+      let serverConfig = self.makeMTLSServerConfigWithCallback(
+        certificatePath: filePaths.serverCert,
+        keyPath: filePaths.serverKey,
+        trustRootsPath: filePaths.trustRoots
+      ) { certificates, promise in
+        let presentedCertificates = certificates.map {
+          try! Certificate(derEncoded: $0.toDERBytes())
+        }
+        #expect(expectedCertificates == presentedCertificates)
+        promise.succeed(
+          .certificateVerified(VerificationMetadata(ValidatedCertificateChain(certificates)))
+        )
+        confirmation.confirm()
+      }
+      // Run the test
```

Comment:
same thing here (and other places):

```suggestion
      }
      
      // Run the test
```